include/openspace/events/eventengine.h

```diff
@@ -105,6 +122,14 @@ class EventEngine {
      */
     void unregisterEventAction(uint32_t identifier);
 
+    /**
+     * Removing registration for a topic/type combination
```

Comment:
Maybe having a line in here describing what happens if you try to unregister a topicid that doesn't exist, or a topicid that is not mapped to the passed type